WeakAuras/AuraEnvironment.lua

```diff
@@ -520,7 +520,8 @@ local overridden = {
   WA_Utf8Sub = WA_Utf8Sub,
   ActionButton_ShowOverlayGlow = WeakAuras.ShowOverlayGlow,
   ActionButton_HideOverlayGlow = WeakAuras.HideOverlayGlow,
-  WeakAuras = FakeWeakAuras
+  WeakAuras = FakeWeakAuras,
+  tsu = Private.TSU.createOrUpdateOrWipe
```

Comment:
I think we should try to avoid creating any more magic globals. I don't think there's any good reason to not just have this be in WeakAuras global?

Reply:
I think the cleanest is to put it into the allstates table via a metatable, that means that it's almost perfectly backwards compatible, as iterating allstates doesn't see the metatable.

So allstates:Update(key, { .... }) (which does CreateOrUpdate, but that's a too long name)
allstates:Remove(key) and maybe allstates:RemoveAll() would be my suggestion.

Reply:
updated PR & examples to do it this way